Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -422,18 +423,31 @@ private let _codeClass = CssSelector.class("code")
 private let codeStyles =
   _codeClass % (
     backgroundColor(.other("#fafafa"))
+      <> color(.other("#24292e"))
       <> fontFamily(["monospace"])
 )
 
 private let inlineCodeStyles =
   Class.pf.inlineCode % (
-    fontFamily(["monospace"])
+    color(.other("#24292e"))
+      <> fontFamily(["monospace"])
       <> padding(topBottom: .px(1), leftRight: .px(5))
       <> borderWidth(all: .px(1))
       <> borderRadius(all: .px(3))
       <> backgroundColor(Color.other("#fafafa"))
 )
 
+private let token = CssSelector.class("token")
+
+private let prismJsTheme =
```

Comment:
Added our own theme here based on GitHub colors. We should explore making the Point-Free theme more light-background friendly.